Code/Editor/ViewportManipulatorController.cpp

```diff
@@ -116,6 +126,14 @@ namespace SandboxEditor
                     }
                     eventType = MouseEvent::Down;
                 }
+
+                m_virtualNormalizedPosition = AZStd::nullopt;
```

Comment:
update: I'm still not sure why this is reset in mouse move and not mouse down, shouldn't this happen only when we've clicked on a manipulator? We also _only_ want this code to run if `SandboxEditor::ManipulatorMouseWrap()` is set to true (makes it easier to disable if there's a problem with the change)

If you look lower down, where we call...

```
InteractionBus::EventResult(
    interactionHandled, AzToolsFramework::GetEntityContextId(), targetInteractionEvent, mouseInteractionEvent);
```

```
if (interactionHandled && SandboxEditor::ManipulatorMouseWrap() && event.m_priority == ManipulatorPriority)
{
    m_virtualNormalizedPosition = AZStd::nullopt;
}
```